test/test.js

```diff
@@ -797,6 +799,34 @@ test('HTTP method is case insensitive', (t) => {
   })
 })
 
+test('form-data should be handled correctly', (t) => {
+  t.plan(3)
+
+  const dispatch = function (req, res) {
+    let body = ''
+    req.on('data', d => {
+      body += d
+    })
+    req.on('end', () => {
+      res.end(body)
+    })
+  }
+
+  const form = new FormData()
+  form._boundary = 'the-boundary'
```

Comment:
I would not rely on this `_boundary` property, it makes the test less reliable.

Reply:
done